### PR TITLE
feat: jsx-a11y/no-access-key, jsx-a11y-no-autofocus, jsx-a11y-no-scope

### PR DESCRIPTION
**Pull request by bitpshr**
This pull request implements the [jsx-a11y/no-access-key](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-access-key.md), the [jsx-a11y-no-autofocus](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-autofocus.md), and the [jsx-a11y-no-scope](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/scope.md) lint rules.

References #341

### CODE REVIEW

File: packages/@romejs/string-markup/parse.ts [hunk 1]
```diff
@@ -61,14 +62,15 @@ tags.set("ol", ["reversed", "start"]);
...
-tagsToOnlyChildren.set("tr", ["td"]);
+tagsToOnlyChildren.set("tr", ["th", "td"]);
 tagsToOnlyChildren.set("ol", ["li"]);
 tagsToOnlyChildren.set("ul", ["li"]);
 
 // Tags that should only be children of other tags
 const tagsToOnlyParent: Map<MarkupTagName, Array<MarkupTagName>> = new Map();
 tagsToOnlyParent.set("tr", ["table"]);
 tagsToOnlyParent.set("td", ["tr"]);
+tagsToOnlyParent.set("th", ["tr"]);
```
Comment (sebmck) [inline packages/@romejs/string-markup/parse.ts:73]: What's the rationale for these changes? The JSX above should never hit the string markup parser.

Comment (bitpshr) [reply packages/@romejs/string-markup/parse.ts]: This was leftover from debugging; removed.

**Comment by sebmck** [comment]
Looks good! Feel free to buffer up future lint rules in the same PR in the future if it's easiest. Thank you!
